debug-reindexing.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+
+function wait_weaviate() {
+  echo "Wait for Weaviate to be ready on $1"
+  for _ in {1..120}; do
+    if curl -sf -o /dev/null localhost:$1/v1/.well-known/ready; then
+      echo "Weaviate is ready"
+      return 0
+    fi
+
+    echo "Weaviate is not ready on $1, trying again in 1s"
+    sleep 1
+  done
+  echo "ERROR: Weaviate is not ready in port ${1} after 120s"
+  exit 1
+}
+
+function shutdown() {
+  echo "Cleaning up ressources..."
+  docker compose -f apps/debug-reindexing-endpoint/docker-compose.yml down --remove-orphans
+  rm -rf apps/weaviate/data* || true
+}
+trap 'shutdown; exit 1' SIGINT ERR
```

Comment:
**suggestion:** 
we can replace that block by `source common.sh` and that will include those 2 func by default and also trap and log the containers in case of any `exit 1`

Reply:
but this one starts it's own docker compose file: `apps/debug-reindexing-endpoint/docker-compose.yml` that is based on `apps/weaviate` docker compose files. I think that `common.sh` won't work